### ISSUE TITLE
Memory leak due to endless requests if request blocked by uMatrix

### ISSUE THREAD

**Issue by Perflyst** [uBlockOrigin/uMatrix-issues#176]
### Prerequisites

- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
    - [x] I also searched the existing issues at <https://github.com/gorhill/uMatrix/issues>
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uMatrix](https://old.reddit.com/r/uMatrix/)
- I tried to reproduce the issue when...
    - [x] uMatrix is the only extension
    - [x] uMatrix with default lists/settings
    - [x] using a new, unmodified browser profile
- [x] I am running the latest version of uMatrix
- [x] I checked the [documentation](https://github.com/gorhill/uMatrix/wiki) to understand that the issue I report is not a normal behavior
- [ ] I used the logger to rule out that the issue is caused by my ruleset

### Description

I have to use some internal system which tries to connect to googleapis.com and only stops if it has successfully reached it. There are thousands of requests in seconds if url is blocked via uMatrix.

After a few hours this results in a quite huge memory leak and browser freezes or if waiting long enough the system goes OOM.

I already tried to answer the requests with a local nginx and self signed certificate but this is not that easily possible due to hardcoded HSTS in Firefox for Google domains.

Maybe the solution is that uMatrix answers the request in any case?

### A specific URL where the issue occurs

I cannot provide on right now, but I will try to build some simple demo site.
Basically anything which re-tries a http request until it succeeds (it will never if uMatrix blocks it)

### Steps to Reproduce

1. Block third party request
2. Open dev tools -> network
3. See the very high connecting rate

### Ruleset

will paste when demo site is up

### Supporting evidence

will paste when demo site is up

### Your environment

* uMatrix version: 1.13.6
* Browser Name and version: Firefox 68
* Operating System and version: Debian 10

**Comment by uBlock-user** [comment]
> After a few hours this results in a quite huge memory leak and browser freezes

So it's browser issue then, how do you this as uMatrix issue ?

> I cannot provide on right now, but I will try to build some simple demo site.
Basically anything which re-tries a http request until it succeeds (it will never if uMatrix blocks it)

There's nothing to investigate here if you cannot provide a URL right now.

**Comment by gorhill** [comment]
> There are thousands of requests in seconds if url is blocked via uMatrix

<https://github.com/uBlockOrigin/uMatrix-issues>:

> Do not post issues regarding broken sites.
